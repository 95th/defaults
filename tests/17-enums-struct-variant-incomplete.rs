use defaults::Defaults;

#[derive(Defaults)]
#[def = "Foo::A { a: 10 }"]
enum Foo {
    A { a: u8, b: usize },
    B { x: u8, y: usize },
}

fn main() {}
