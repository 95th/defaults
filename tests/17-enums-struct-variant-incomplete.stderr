error[E0063]: missing field `b` in initializer of `Foo`
 --> $DIR/17-enums-struct-variant-incomplete.rs:4:9
  |
4 | #[def = "A { a: 10 }"]
  |         ^^^^^^^^^^^^^ missing `b`
